app/controllers/ticket_scannings_controller.rb

```diff
@@ -0,0 +1,17 @@
+class TicketScanningsController < ApplicationController
+  before_action :authenticate_user!
+  load_and_authorize_resource
+
+  def create
+    @physical_ticket = PhysicalTicket.find_by_token(params[:token])
+
+    if @physical_ticket.present?
+      @ticket_scanning = TicketScanning.create(physical_ticket: @physical_ticket)
+      redirect_to conferences_path,
+                  notice: 'Thanks! Your ticket is scanned successfully.'
```

Comment:
I would remove the `thanks` as QR would read by admins. Also, it is not `your` ticket` but `the` ticket. And would be nice to add the token in the message.